mysql-test/suite/encryption/t/percona_redo_default_key.test

```diff
@@ -0,0 +1,158 @@
+--source include/have_debug.inc
+
+let $MYSQLD_BASEDIR= `select @@basedir`;
+--mkdir $MYSQL_TMP_DIR/datadir1
+
+let $MYSQLD_DATADIR1 = $MYSQL_TMP_DIR/datadir1/data;
+--source include/shutdown_mysqld.inc
+
+--let BOOTSTRAP_SQL=$MYSQL_TMP_DIR/boot.sql
+--let KEYRING_FILE_PATH=$MYSQL_TMP_DIR/mysecret_keyring
+--let KEYRING_DATA="--keyring_file_data=$KEYRING_FILE_PATH"
+--let BOOTSTRAP_LOG=$MYSQL_TMP_DIR/bootstrap.log
+
+--echo # create bootstrap file
+write_file $BOOTSTRAP_SQL;
+CREATE DATABASE test;
+EOF
+
+--let BOOTSTRAP_CMD = $MYSQLD_CMD --initialize-insecure --basedir=$MYSQLD_BASEDIR --datadir=$MYSQLD_DATADIR1 --init-file=$BOOTSTRAP_SQL $KEYRING_PLUGIN_OPT $KEYRING_PLUGIN_EARLY_LOAD $KEYRING_DATA --innodb_redo_log_encrypt=KEYRING_KEY > $BOOTSTRAP_LOG 2>&1
+
+--exec $BOOTSTRAP_CMD
+
+--remove_file $BOOTSTRAP_LOG
+--remove_file $MYSQLD_DATADIR1/ib_logfile0
+--remove_file $MYSQLD_DATADIR1/ib_logfile1
+
+--let $test_error_log=$MYSQL_TMP_DIR/error_log.err
+
+# We crash before percona_redo:0 gets rotated to percona_redo-UUID:1 (it gets rotated when UUID becomes available)
+
+--error 137
+--exec $MYSQLD_CMD --loose-skip-log-bin --skip-log-slave-updates --datadir=$MYSQLD_DATADIR1 $KEYRING_DATA $KEYRING_PLUGIN_OPT $KEYRING_PLUGIN_EARLY_LOAD --log-error=$test_error_log --innodb_redo_log_encrypt=KEYRING_KEY --debug=+d,crash_before_redo_key_is_rotated
```

Comment:
just use d,crash_before..
It might create problems on other operating systems

Reply:
done